### PR TITLE
504 & 505 patient select UI updates and hiding empty state

### PR DESCRIPTION
**Pull request by avirgulto**
## Describe your changes

Updates to the patient select button, hiding fields and buttons when no patient is selected, added in a 'empty' state vs N/A when there is no data, rename some of the button names, disabled the 'open in-progress forms' button if no form is selected, and change button for open developer console instead of +.

To test:
Load the request generator, see all visual changes on initial load when no patient is selected. Select a patient without a medication. See the buttons and information show, with the appropriate buttons disabled. Then load the patient with medication and see the other buttons enabled as well as some more information filled in. Then go through and get an in progress form. Open up an in progress form and see the 'open in-progress form' button is enabled. 

## Issue ticket number and Jira link

https://jira.mitre.org/browse/REMS-504 and https://jira.mitre.org/browse/REMS-505 

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

**Review by jtquach1 — CHANGES_REQUESTED** [review]
I think this looks good to me, with some minor comments. Should we include tests or is that something @afeinman-mitre is working on that we could add later on?

File: src/containers/RequestBuilder.js [hunk 1]
```diff
@@ -448,6 +448,9 @@ export default class RequestBuilder extends Component {
                         />
                     </div> */}
 
+                    <br />
```
[the remark this answers is not in the document]
Comment (avirgulto) [reply src/containers/RequestBuilder.js:429]: Yeah in the future when we update the overall structure we can use better spacing!

**Comment by avirgulto** [comment]
> I think this looks good to me, with some minor comments. Should we include tests or is that something @afeinman-mitre is working on that we could add later on?

Yeah i think once the testing is merged into develop and stuff, we can definitely add a small test for the new workflow of hidden and disabled things! I'll make a ticket in the backlog for this!

**Comment by plarocque4** [comment]
I like how the prefetched area disappears when the in-progress form is selected. Could we do the same with the In Progress Form area? Could it only be visible with a selected in progress form and hide otherwise?

**Review by plarocque4 — APPROVED** [review]
Looks good

**Pull request merged**